src/js/core/widget/core/Card.js

```diff
@@ -179,7 +179,6 @@
 			 * @protected
 			 */
 			prototype._destroy = function () {
```

Comment:
I think you can remove whole method if this method does nothing.

Reply:
Done